experimental/packages/opentelemetry-sdk-metrics-base/src/MeterProvider.ts

```diff
@@ -48,16 +45,24 @@ export class MeterProvider {
         api.diag.warn('A shutdown MeterProvider cannot provide a Meter')
         return metrics.NOOP_METER;
     }
+    // TODO: add schemaUrl to id?
```

Comment:
What does the spec say about this? I thought the schema URL was considered part fo the identity?

Reply:
I dug into the previous discussion on this https://github.com/open-telemetry/opentelemetry-js/pull/2529#discussion_r739290425. So I'd believe the schemaUrl should be part of the identity, yes. I'll update this.

Reply:
Extracted this part of change to https://github.com/open-telemetry/opentelemetry-js/pull/2666 as it is not the essential part of the metric streams.

Reply:
I don't want this small thing to block the whole PR which is mostly unrelated so do you want to remove this change from this PR so it can be handled in #2666 or do you want to keep what you have and we can iterate on it in #2666 if needed? Either is fine with me, we just need to make sure we're moving forward.

Reply:
Let me know what you want to do and I'll try to get this merged tomorrow

Reply:
Thanks for the feedback! I've removed these changes and updated the tests :).